tree/ntuple/v7/src/RNTupleProcessor.cxx

```diff
@@ -181,9 +181,13 @@ ROOT::Experimental::RNTupleChainProcessor::RNTupleChainProcessor(const std::vect
    if (fNTuples.empty())
       throw RException(R__FAIL("at least one RNTuple must be provided"));
 
+   fInnerNEntries.assign(ntuples.size(), kInvalidNTupleIndex);
+
    fPageSource = Internal::RPageSource::Create(fNTuples[0].fNTupleName, fNTuples[0].fStorage);
    fPageSource->Attach();
 
+   fInnerNEntries.at(0) = fPageSource->GetNEntries();
```

Comment:
This doesn't need to be a checked access since you are ensuring `ntuples` has at least 1 element a few lines above